### PR TITLE
Update slovak translation

### PR DESCRIPTION
**Pull request by erciccione**
Updates made by @fero-sk (see [#1377 comment](https://github.com/monero-project/monero-gui/issues/1377#issuecomment-389219793))

### CODE REVIEW

File: translations/monero-core_sk.ts [hunk 1]
```diff
@@ -1022,12 +1022,12 @@
...
         <location filename="../pages/Settings.qml" line="680"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;a href=&apos;#&apos;&gt; (Kliknutím zmeňte)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="682"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Nastavte novú výšku obnovenia</translation>
```
Comment (erciccione) [inline translations/monero-core_sk.ts]: missing `:`

File: translations/monero-core_sk.ts [hunk 2]
```diff
@@ -1117,17 +1117,17 @@
...
         <location filename="../pages/Settings.qml" line="657"/>
         <source>Wallet name: </source>
-        <translation type="unfinished"></translation>
+        <translation>Názov peňaženky: </translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="671"/>
         <source>Wallet creation height: </source>
-        <translation type="unfinished"></translation>
+        <translation>Výška vytvorenia peňaženky</translation>
```
Comment (erciccione) [inline translations/monero-core_sk.ts]: missing `:` + whitespace

**Comment by fero-sk** [comment]
I'll check it. And will try to translate the rest (following the czech files).

File: translations/monero-core_sk.ts [hunk 3]
```diff
@@ -784,102 +784,102 @@
...
         <location filename="../pages/Receive.qml" line="614"/>
         <source>confirmations</source>
-        <translation type="unfinished"></translation>
+        <translation>potvrdení</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="617"/>
         <source>confirmation</source>
-        <translation type="unfinished"></translation>
+        <translation>potvrdení</translation>
```
Comment (MaxXor) [inline translations/monero-core_sk.ts]: `potvrdenie`

File: translations/monero-core_sk.ts [hunk 4]
```diff
@@ -980,24 +980,24 @@
...
         <location filename="../pages/Settings.qml" line="207"/>
         <source>Daemon mode</source>
-        <translation type="unfinished"></translation>
+        <translation>Režim démona</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="240"/>
         <source>Bootstrap node</source>
-        <translation type="unfinished"></translation>
+        <translation>Bootstrap režim</translation>
```
Comment (MaxXor) [inline translations/monero-core_sk.ts]: `Bootstrap uzol`

File: translations/monero-core_sk.ts [hunk 5]
```diff
@@ -1057,52 +1057,52 @@
...
         <location filename="../pages/Settings.qml" line="334"/>
         <source>Manage Daemon</source>
-        <translation type="unfinished"></translation>
+        <translation>Spravovať démona</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="438"/>
         <source>Show advanced</source>
-        <translation type="unfinished"></translation>
+        <translation>Ukázať pokričilé</translation>
```
Comment (MaxXor) [inline translations/monero-core_sk.ts]: `pokričilé` -> `pokročilé`

File: translations/monero-core_sk.ts [hunk 6]
```diff
@@ -1214,29 +1221,29 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="93"/>
         <source>Shared RingDB</source>
-        <translation type="unfinished"></translation>
+        <translation>Zdieľaná databáza okruhov</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="97"/>
         <source>This page allows you to interact with the shared ring database. This database is meant for use by Monero wallets as well as wallets from Monero clones which reuse the Monero keys.</source>
-        <translation type="unfinished"></translation>
+        <translation>Táto stránka Vám umožní komunikovať so zdieľanou databázou okruhov. Táto databáza je určená na požitie s Monero peňazenkami a taktiež s peňaženkami Monero klonov, ktoré opätovne používajú Monero kľúče.</translation>
```
Comment (MaxXor) [inline translations/monero-core_sk.ts]: `požitie` -> `použitie`

**Comment by erciccione** [comment]
@MaxXor i implemented changes according to erciccione/monero-core#5

**Comment by erciccione** [comment]
@MaxXor @fero-sk We have no time, release could happen in moments. Rebasing now

**Comment by erciccione** [comment]
@luigi1111 another one ready

**Pull request merged**